5.5/Dockerfile

```diff
@@ -68,6 +68,9 @@ RUN mkdir -p /var/lib/mysql /var/run/mysqld \
 # ensure that /var/run/mysqld (used for socket and lock files) is writable regardless of the UID our mysqld instance ends up having at runtime
 	&& chmod 777 /var/run/mysqld
 
+# default root to listen for connections from anywhere
+ENV MYSQL_ROOT_HOST %
```

Comment:
This is going to cause issues with `file_env` (`-e MYSQL_ROOT_HOST_FILE=...` will lead to `error: both MYSQL_ROOT_HOST and MYSQL_ROOT_HOST_FILE are set (but are exclusive)`), so needs to be specified as the default in the script instead (via the second argument to `file_env`).